L1Trigger/L1THGCalUtilities/python/concentrator.py

```diff
@@ -90,3 +90,28 @@ def create_mixedfeoptions(process, inputs,
             )
     return producer
 
+
+def create_custom(process, inputs,
+                            stcSize=supertc_conc_proc.stcSize,
+                            type_energy_division=supertc_conc_proc.type_energy_division,
+                            fixedDataSizePerHGCROC=supertc_conc_proc.fixedDataSizePerHGCROC,
+                            triggercells=best_conc_proc.NData,
+                            threshold_silicon=threshold_conc_proc.threshold_silicon,  # in mipT
+                            threshold_scintillator=threshold_conc_proc.threshold_scintillator,  # in mipT
+                            Method = custom_conc_proc.Method
+                            ):
```

Comment:
Shouldn't `custom_conc_proc` be used for the default values?

Reply:
yes, thanks